### PR TITLE
Update force training example

### LINKED ISSUE #243: Make torchani's training setup identical to neurochem
After a thorough comparison between Neurochem and Torchani, I have found a few differences as explained briefly in the following:

- [x] NC skips the weight decay for the first layer of each atomic network, even if the weight decay parameter is provided in the input. This was the main source of difference in training performance.(#275)
- [x] In NC the weights are initialized to He Kaiming normal and the biases to zero. (#245, #275)
- [x] Loss in NC is scaled by sqrt(natoms). (#275)
- [x] NC uses AdamW optimizer to update the weights and SGD for the biases. (#275)
- [x] There were a couple of discrepancies between theory and the implementation of AdamW in Torchani (#261).
- [x] NC scale the gradients of atomic networks by the number of atoms in that batch, and compute the gradient from SumSquaredError instead of MeanSquaredError. (will be fixed in Neurochem)
- [ ] NC reshuffle the batches after every epoch (tracked in #248, no much improvement as tested, low priority)

### PR DESCRIPTION
**Pull request by farhadrgh**
This PR fixes #243

### CODE REVIEW

File: examples/nnp_training_ignite.py [hunk 1]
```diff
@@ -59,10 +60,9 @@
 # Instead of manually specifying hyperparameters as in :ref:`training-example`,
 # here we will load them from files.
 const_file = os.path.join(path, '../torchani/resources/ani-1x_8x/rHCNO-5.2R_16-3.5A_a4-8.params')  # noqa: E501
-sae_file = os.path.join(path, '../torchani/resources/ani-1x_8x/sae_linfit.dat')  # noqa: E501
 consts = torchani.neurochem.Constants(const_file)
 aev_computer = torchani.AEVComputer(**consts)
-energy_shifter = torchani.neurochem.load_sae(sae_file)
+energy_shifter = torchani.neurochem.load_sae(None)
```
Comment (zasdfgbnm) [inline examples/nnp_training_ignite.py]: What does `load_sae(None)` do?

Comment (farhadrgh) [reply examples/nnp_training_ignite.py]: oops! that was a mistake! fixed in https://github.com/aiqm/torchani/pull/283/commits/f6346846ad90f003e1c789d16586327b2768a643

File: examples/nnp_training_force.py [hunk 2]
```diff
@@ -33,12 +34,7 @@
 ShfA = torch.tensor([9.0000000e-01, 1.5500000e+00, 2.2000000e+00, 2.8500000e+00], device=device)
 num_species = 4
 aev_computer = torchani.AEVComputer(Rcr, Rca, EtaR, ShfR, EtaA, Zeta, ShfA, ShfZ, num_species)
-energy_shifter = torchani.utils.EnergyShifter([
-    -0.600952980000,  # H
-    -38.08316124000,  # C
-    -54.70775770000,  # N
-    -75.19446356000,  # O
-])
+energy_shifter = torchani.utils.EnergyShifter([None])
```
Comment (zasdfgbnm) [inline examples/nnp_training_force.py]: I think it should be `EnergyShifter(None)`

Comment (farhadrgh) [reply examples/nnp_training_force.py]: fixed in https://github.com/aiqm/torchani/pull/283/commits/2125d7fc80a8abf822c89c3f0e510ab4eea3abc2
